### ISSUE TITLE
[Cognitive - Language - Conversational SA (runtime)] API Async Update

### ISSUE THREAD

**Issue by markweitzel** [Azure/azure-rest-api-specs#19762]
* Service name: Cognitive Service
* Key contact for the review: Ranran Wang ([rawan@microsoft.com](mailto:rawan@microsoft.com)) , Kevin Endres ([keendres@microsoft.com](mailto:keendres@microsoft.com))
* Whether this is a new or existing API: Existing API
* Whether or not your service already been deployed: It has deployed
* A brief description of the material to be reviewed/changed : The changes requests to review is for Language Conversation API Async Update. Adding a new feature to support Conversational Sentiment Analysis
* A link to the PR/issue in the azure-rest-api-specs (public)  or azure/azure-rest-api-specs-pr (private) repo
The PR link is [[Cognitive - Language API 2022-10-01-Preview] Conversation API Async Update - Conversational Sentiment Analysis by jessicawrr · Pull Request #19742 · Azure/azure-rest-api-specs (github.com)](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F19742&data=05%7C01%7Cmarkweitzel%40microsoft.com%7Ccadeccc9b104459bf2d908da63c5ac60%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637931997894706655%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=aaUpJ78G3ViOh62MS0FQMn6KZd5nnTPoWewV2tdzfzo%3D&reserved=0)

**Comment by jessicawrr** [comment]
Key scenarios:
Post conversation/call analytics for contact center supervisors to identify agent coaching opportunities through sentiment insights, including Turn by turn sentiment to enable the visualization of sentiment trending over conversations

Design Doc: https://microsoft.sharepoint-df.com/:w:/r/teams/LanguageCustomization/_layouts/15/doc2.aspx?sourcedoc=%7B08AF0E0D-E55C-4D20-B9D4-CC3E3C3B355C%7D&file=Conversational%20Sentiment%20Design%20Spec.docx&action=default&mobileredirect=true&DefaultItemOpen=1&share=IQENDq8IXOUgTbnUzD48OzVcAda1-jHX0hgeHqr9fgOqVJ4

Change log:

- Introduce a new analysis task on a collection of conversations for sentiment analysis.
- Introduce a new analysis task result type on a collection of conversations for sentiment analysis.

**Comment by mikekistler** [comment]
Notes from API review meeting, 7/26/2022:

- Need to work out the api versioning mechanics. Heath to provide guidance to the team.
- Lots of discussion on "mixed" confidence scores. RanRan to add examples to clarify the service behavior.
- Should sentence returns be required or optional?
- Should the concept of sentences be used or should be it "turns"?
- Is the `showStats` parameter really supported? No ... remove from design doc.

**Comment by jessicawrr** [comment]
I've addressed all issues assign to me in the meeting notes. Below are the details:

- Lots of discussion on "mixed" confidence scores. RanRan to add examples to clarify the service behavior.
**[RR]** Example has been added and PR has updated
- Should sentence returns be required or optional?
- Should the concept of sentences be used or should be it "turns"?
**[RR]** Above 2 are related to each other. After offline discussed with Kevin and Mikael. We agree to drop the object of the sentence. I've updated the PR. 
- Is the showStats parameter really supported? No ... remove from design doc.
**[RR]** Removed from the design doc.
